imports/plugins/core/checkout/client/components/completedShopOrders.js

```diff
@@ -10,16 +9,24 @@ import CompletedOrderItem from "./completedOrderItem";
  * @property {String} shopName - The name of the shop
  * @property {Array} items - an array of individual items for this shop
  * @property {Function} handleDisplayMedia - A function for displaying product images
+ * @property {boolean} isProfilePage - Checks if current page is Profile Page
  * @return {Node} React node containing the break down of the order by Shop
  */
-const CompletedShopOrders = ({ shopName, items, handleDisplayMedia, shippingMethod }) => {
+const CompletedShopOrders = ({ shopName, items, handleDisplayMedia, shippingMethod, isProfilePage }) => {
+  const shippingName = isProfilePage ? (
+    <span>
+      <strong>
+        {shippingMethod.label}
+      </strong>{shippingMethod.deliveryDate && <span> - estimated delivery {shippingMethod.deliveryDate}</span>}
+    </span>
+  ) : `${shippingMethod.carrier} - ${shippingMethod.label}`;
   return (
     <div className="order-details-shop-breakdown">
       {/* This is the left side / main content */}
       <div className="order-details-info-box">
         <div className="store-detail-box">
           <span className="order-details-store-title">{shopName}</span>
-          <span className="order-details-shipping-name">{shippingMethod.carrier} - {shippingMethod.label}</span>
+          <span className="order-details-shipping-name">{shippingName}</span>
```

Comment:
Why was this changed?

Reply:
`shippingName` varies depending on whether the current page is profile page. It is defined in  line 16.